htdocs/core/class/html.formmail.class.php

```diff
@@ -1004,6 +1004,8 @@ public function get_form($addfileaction = 'addfile', $removefileaction = 'remove
 				$out .= '</td>';
 				$out .= '<td>';
 
+				$this->withaiprompt = 'html';
```

Comment:
This must not be always set.

It is the caller that make the new FormMail() that will decide if he want to set ->withaiprompt or ->withlayout.
So move this at the parent caller place...